tests/schema_advanced.py

```diff
@@ -94,7 +94,7 @@ class Cell(dj.Manual):
 class InputCell(dj.Manual):
     definition = """  # a synapse within the slice
     -> Cell
-    (input)-> Cell(cell)
+    -> Cell.proj(input="cell")
```

Comment:
Same as above

Reply:
While this table uses the new syntax, many existing tests use the old syntax.  They don't show in the diff because they remained unchanged.